src/graderPublic/java/h13/util/StudentLinks.java

```diff
@@ -471,7 +471,14 @@ public enum GameSceneMethodLink implements ClassMethodLink {
                 () -> GameScene.class.getDeclaredMethod("init")
             ))),
             INIT_GAMEBOARD_METHOD(BasicMethodLink.of(Assertions.assertDoesNotThrow(
-                () -> GameScene.class.getDeclaredMethod("initGameboard")
+                () -> {
+                    // Typo in initial student template
+                    try {
+                        return GameScene.class.getDeclaredMethod("initGameboard");
+                    } catch (NoSuchMethodException e) {
+                        return GameScene.class.getDeclaredMethod("initGameBoard");
+                    }
+                }
             ))),
```

Comment:
This is not very clean and will certainly lead to confusion as it does not provide proper Error Messages if students tamper with the method signature.
Please use the following snippet instead:
```java
INIT_GAMEBOARD_METHOD(Assertions3.assertMethodExists(
    BasicTypeLink.of(GameScene.class),
    MatcherUtils.or(
        identical("initGameboard"),
        identical("initGameBoard")
    )
)),
```

Reply:
Should be implemented now.